startupScript.py

```diff
@@ -199,7 +210,10 @@ def restart_excepthook(etype, value, tb):
 	os.remove(logPML)
 
 # Randomly select what type of build we are going to profile
-testBuild = random.choice(buildList)
+if useRandomBuild or not cohort:
+	testBuild = random.choice(buildList)

```

Comment:
@squarewave since we're setting `cohort = None` I think this if check would always execute, right?